plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -26,6 +26,7 @@ global proc string getMayaUsdString(string $key)
 global proc mayaUSDRegisterStrings()
 {
     register("kButtonSave", "Save");
+    register("kButtonSet", "Set");
     register("kButtonCancel", "Cancel");
     register("kCreateUsdStageFromFile", "Create USD Stage from File");
```

Comment:
I think this file can be reverted

Reply:
Why to revert it? kButtonSet and kMenuSetAs are used in mayaUsd_layerEditorFileDialogs.mel

Reply:
OH for some reason I thought you were using the Maya resources in MEL.